### ISSUE TITLE
`fromSQL` does not parse input when time zone is `UTC`

### ISSUE THREAD

**Issue by aar0nr** [moment/luxon#380]
This works:

```js
DateTime.fromSQL('2018-12-10 16:00:00 America/Denver', { setZone: true })
```

But when the time zone is UTC it results in an unparsable date:

```js
DateTime.fromSQL('2018-12-10 16:00:00 UTC', { setZone: true })
```

```js
{
  reason: "unparsable",
  explanation: "the input \"2018-12-10 16:00:00 UTC\" can't be parsed as SQL"
}
```

**Comment by icambron** [comment]
fixed in 9e97c08, will go into the next release

**Comment by icambron** [comment]
This was released in 1.8.3
